benchmark/async_hooks/http-server.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+const common = require('../common.js');
+
+const bench = common.createBenchmark(main, {
+  asyncHooks: ['enabed', 'disabled', 'none'],
+  c: [50, 500]
+});
+
+function main({ asyncHooks, c }) {
+  if (asyncHooks !== 'none') {
+    const hook = require('async_hooks').createHook({
+      init() {},
+      before() {},
+      after() {},
+      destroy() {},
+      promiseResolve() {}
```

Comment:
AFAIK there's a performance difference in case only `init`, `before` and `after` are used. Should this be a separate option?

Reply:
Added some different variants of option `asyncHooks`: `all`, `init`, `before` and `after`.